.tool-versions

```diff
@@ -2,6 +2,7 @@
 
 actionlint 1.7.7
 cosign 2.5.0
+diffoci 0.1.6
```

Comment:
The [nightly tool updating job](https://github.com/ericcornelissen/js-regex-security-scanner/blob/dbe4b993186f28cdbf35f6a4947744fbc5470797/.github/workflows/nightly.yml#L10-L38) should be updated to add the [`diffoci` plugin](https://github.com/ericcornelissen/asdf-diffoci) too, following [it's documentation](https://github.com/ericcornelissen/tool-versions-update-action/blob/3743d6eaf1c5152fa30bbed750d97fff007fb560/pr/README.md#usage).

Reply:
Resolved with c0aa82e7922218b657f50b2aae357f0ded51dc69